src/manifest/index.ts

```diff
@@ -0,0 +1,19 @@
+export type Asset = string;
+
+export type Manifest = {
+  publicPath: string;
+  assets: {
+    [id: string]: Asset[];
+  };
+};
+
+export const getAssetUrlsFromId = (
+  manifest: Manifest,
+  id: string
+): Asset[] | undefined => {
+  if (!manifest.assets[id]) {
+    return;
+  }
+
+  return manifest.assets[id].map(asset => `${manifest.publicPath}${asset}`);
```

Comment:
Should we handle `publicPath` being undefined (for whatever reason)?
Also if `asset` is `./bla` and `publicPath` is `/` is we end up with `/./bla` 🤔 Should we do something about it?

Reply:
This is why I made the `publicPath` configurable, which is what both `html-webpack-plugin` and `webpack-manifest-plugin` provide. The html plugin does minimal handling, but not for the cases you mentioned, while the logic here is the same as the manifest plugin.